src/main/flight/servos.c

```diff
@@ -376,10 +376,10 @@ void writeServos(void)
         break;
     }
 
-    // Two servos for SERVO_TILT, if enabled
+    // One servo for SERVO_TILT, if enabled
     if (feature(FEATURE_SERVO_TILT) || currentMixerMode == MIXER_GIMBAL) {
         updateGimbalServos(servoIndex);
-        servoIndex += 2;
+        servoIndex += 1;
```

Comment:
What exactly is this line doing? Why has it been changed to +1 instead of +2?

Reply:
just one servo

Reply:
So how does it effect things if you want 2 serves.

Reply:
this code ancient. the original idea was to control one servo per axis, pitch and roll. nicecrash was only looking to stabilize his gopro as shown on youtube - so i guess this should work for most cinema pilots.
it might be a breaking change for people using this with two servos but i bet the risk is super low 😸